### PR TITLE
To-Do List: add & remove

### PR DESCRIPTION
**Pull request by arnrdn**
Hello!

In this PR I:
1. Added delete all checked items feature
2. Added delete one item feature
3. Added add one item feature
4. List item change its background color and trash can icon appears on click
5. When you change a task it immediately saves the new value in the local storage

Thank you for reviewing my code!

### CODE REVIEW

File: src/addRemove.js [hunk 1]
```diff
@@ -15,12 +16,45 @@ function createTask(name) {
...
-  render();
+  save(tasks);
+  render(tasks);
+  document.location.reload();
 }
 
-export { newTaskForm, addTask };
+function deleteItem() {
+  const listItems = document.querySelectorAll('li');
+  listItems.forEach((item, index) => {
```
Comment (Dipeshtwis) [inline src/addRemove.js]: - [ ] I would recommend using `es6` syntax as it is a project requirement which helps you in writing more clean code using ES6 syntax.

File: src/interactive.js [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
     selectedTask.complete = e.target.checked;
-    save();
-    render();
+    save(tasks);
+    render(tasks);
   }
 }
+
+function check() {
+  const checkbox = document.querySelectorAll('input[type=checkbox]');
```
Comment (Dipeshtwis) [inline src/interactive.js]: - [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.

File: src/localStorage.js [hunk 3]
```diff
@@ -1,8 +1,21 @@
...
+function save(tasks) {
   localStorage.setItem(taskKey, JSON.stringify(tasks));
 }
 
-export { tasks, taskKey, save };
\ No newline at end of file
+function removeFromStorage(id) {
+  tasks.forEach((task, index) => {
+    if (task.id === id) {
+      tasks.split(index, 1);
```
Comment (Dipeshtwis) [inline src/localStorage.js]: - [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.

File: src/removeAll.js [hunk 4]
```diff
@@ -0,0 +1,10 @@
+import { save } from './localStorage.js';
+import { render } from './renderTask.js';
+
+export default function removeAllCheckedRender() {
+  let tasks = JSON.parse(localStorage.getItem('tasks')) || [];
+  tasks = tasks.filter((task) => !task.complete);
```
Comment (Dipeshtwis) [inline src/removeAll.js]: - [ ] Here as well. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.

File: src/renderTask.js [hunk 5]
```diff
@@ -18,11 +16,37 @@ function render() {
...
+    const trashBtn = taskItem.querySelector('.delete');
+    trashBtn.id = task.id;
     tasksContainer.appendChild(taskItem);
   });
 }
 
-export { render, tasks, tasksContainer };
\ No newline at end of file
+function focus(e) {
+  const clicked = e.target;
```
Comment (Dipeshtwis) [inline src/renderTask.js]: - [ ] Here as well and all the places. I would recommend using `es6` syntax as it is a project requirement that helps you in writing more clean code using ES6 syntax.

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @rdnrn, 

:+1: This is good to make it this far, your project is really nice, congratulations on the good work you have done so far.
:+1: The project is perfectly working.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️

**_Check the comments under the review_**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- N.A.

Cheers and Happy coding!👏👏👏
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**_Dipesh Kumar_**


------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project(check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by EricMbouwe — APPROVED** [review]
### Hi @rdnrn, 

# Status: Approved ✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**